packages/ses/src/global-object.js

```diff
@@ -15,27 +35,18 @@ import { constantProperties, universalPropertyNames } from './whitelist.js';
  * @param {Object} newGlobalPropertyNames
  * @param {Function} makeCompartmentConstructor
  * @param {Object} compartmentPrototype
- * @param {Object} [options]
- * @param {Array<Transform>} [options.globalTransforms]
- * @param {(Object) => void} [options.markVirtualizedNativeFunction]
+ * @param {(string, Object?) => any} safeEvaluate
+ * @param {(Object) => void} markVirtualizedNativeFunction
  */
-export const initGlobalObject = (
+export const initGlobalObjectProperties = (
   globalObject,
   intrinsics,
   newGlobalPropertyNames,
   makeCompartmentConstructor,
   compartmentPrototype,
-  { globalTransforms, markVirtualizedNativeFunction },
+  safeEvaluate,
+  markVirtualizedNativeFunction,
```

Comment:
This is a fair drive-by simplification given that all the arguments are positional+optional. It might be more robust to use a named argument bag, even when TSC watches our back.